src/app/public/src/modules/tutorial/tutorial-summary.component.scss

```diff
@@ -6,13 +6,13 @@
   text-align: center;
   margin: 0 0 20px;
 
-  @media (min-width: $stache-screen-sm) {
+  @media (min-width: $stache-screen-sm-min) {
     font-size: 19.5px;
   }
 
   &,
   p {
-    color: $stache-color-gray;
+    color: $stache-text-color-deemphasized;
```

Comment:
This color isn't enough of a contrast with the well.   fails on the http://webaim.org/resources/contrastchecker/

`#686c73` - font color
`#eeeeef` - background color